standalone-node/provisioning_scripts/install-os.sh

```diff
@@ -437,50 +437,66 @@ EOT
 boot_order_chage_to_disk() {
     echo -e "${BLUE}Changing the Boot order to disk!!${NC}" 
 
+    boot_entry="Emt"
+    efiboot_part=1
+    EFI=""
+
+    # Delete the pile up Ubuntu/Emt partitions from BIOS bootMenu
+    for bootnumber in $(efibootmgr | grep -iE "emt|Ubuntu" | awk '{print $1}' | sed 's/Boot//;s/\*//'); do
```

Comment:
Unquoted variables can break if boot numbers contain spaces. Quote variables, and handle empty cases.